config/core/300-resources/service.yaml

```diff
@@ -58,6 +58,86 @@ spec:
                 items:
                   type: object
                   x-kubernetes-preserve-unknown-fields: true
+                  properties:
+                    revisionName:
+                      type: string
+                      description: |
+                        A specific revision to which to send this portion
+                        of traffic.
+                        This is mutually exclusive with configurationName.
+                        +optional
+                    configurationName:
+                      type: string
+                      description: |
+                        ConfigurationName of a configuration to whose latest revision we will send
+                        this portion of traffic. When the "status.latestReadyRevisionName" of the
+                        referenced configuration changes, we will automatically migrate traffic
+                        from the prior "latest ready" revision to the new one.  This field is never
+                        set in Route's status, only its spec.
+                        This is mutually exclusive with RevisionName.
+                        +optional
```

Comment:
Does it make sense to drop these `+optional`?

Reply:
It was my understanding that you could have only one of `revisionName` OR `configurationName`. Unless you are saying that `configurationName` can never be in `traffic`. 
I copied the comments from the API. 
OR are you just saying that it may be too much api specific info?

Reply:
I was referring to having the literal `+optional` in the description. The prior text seems clear enough

Reply:
I can remove it.